pkg/fieldpath/paved.go

```diff
@@ -38,15 +38,15 @@ func (e errNotFound) IsNotFound() bool {
 // index was out of bounds in an array.
 func IsNotFound(err error) bool {
 	cause := errors.Cause(err)
-	_, ok := cause.(interface {
+	_, ok := cause.(interface { //nolint: errorlint // Skip errorlint for interface type
```

Comment:
Is there no way to use `any` here as well?

Reply:
`any` is only explicitly interoperable with `interface{}`, here it is interface with function definitions

Reply:
Do you know if there is a plan to make it available for use in these cases as well, or are we stuck with `interface{}` for the foreseeable future?